apollo-gradle-plugin-incubating/src/test/kotlin/com/apollographql/apollo/gradle/test/UpToDateTests.kt

```diff
@@ -72,6 +72,27 @@ class UpToDateTests {
     File(dir, "build.gradle").writeText(text.replace(apolloBlock, ""))
   }
 
+  fun `setting singularizeTypes in the build script re-generates the CustomType class`(dir: File) {
+    val apolloBlock = """
+      apollo {
+        singularizeTypes = false
+      }
+    """.trimIndent()
+
+    File(dir, "build.gradle").appendText(apolloBlock)
+
+    val result = TestUtils.executeTask("generateApolloSources", dir)
+
+    // modifying the customTypeMapping should cause the task to be out of date
+    // and the task should run again
```

Comment:
Bad copy/paste. Also this method is unused at the moment. Either call it from `complete test()` or just remove it alltogether, I don't think we need up-to-date checks for each and every parameter as they all work the same.

Reply:
Good call, removed this test altogether.